File: python/461-Hamming-Distance.py
class Solution(object):
    def hammingDistance(self, x, y):
	"""
	:type x: int
	:type y: int
	:rtype: int
	"""
        z=x^y
        ans=0
        for i in range(0,32):
            if z&(1<<i):
                ans+=1
        return ans

File: js/461-Hamming-Distance.js
/**
 *  * @param {number} x
 *   * @param {number} y
 *    * @return {number}
 *     */
var hammingDistance = function(x, y) {
    let z = x ^ y;
    let ans = 0;
    for (let i = 0; i < 32; i++) {
        if (z & (1 << i)) ans++;
    }
    return ans;

};
